src/psfmachine/ffi.py

```diff
@@ -0,0 +1,924 @@
+"""Subclass of `Machine` that Specifically work with FFIs"""
+import os
+import numpy as np
+
+import matplotlib.pyplot as plt
+import matplotlib.colors as colors
+
+from astropy.io import fits
+from astropy.stats import SigmaClip
+from astropy.time import Time
+from astropy.wcs import WCS
+from photutils import Background2D, MedianBackground, BkgZoomInterpolator
+
+# from . import PACKAGEDIR
+from .utils import do_tiled_query
+from .machine import Machine
+from .version import __version__
+
+__all__ = ["FFIMachine"]
+
+
+class FFIMachine(Machine):
+    """
+    Subclass of Machine for working with FFI data. It is a subclass of Machine
+    """
+
+    def __init__(
+        self,
+        channel=1,
+        quarter=5,
```

Comment:
Kepler uses 
* "Quarters", "Modules", "Channels"
K2 uses 
* "Campaigns", "Modules", "Channels"
TESS uses
* "Sectors", "Cameras", "CCDs"
It would be nice if these keywords were a little flexible. For K2, there is no "quarter".